### PR TITLE
fix: strip bom from processed assets (#487)

### PR DESCRIPTION
**Pull request by ahmedGomaa90**
## I'm submitting a...

```
[x] Bug Fix
[ ] Feature
[ ] Other (Refactoring, Added tests, Documentation, ...)
```

## Checklist

fix: strip BOM from processed assets (#487)

## Description

use [strip-bom](https://github.com/sindresorhus/strip-bom) package to remove byte order mark from the string of processed assets, and add it to dependencies in package.json

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by dherges** [comment]
Hi @ahmedGomaa90,

thaks for putting up the PR! I am going to look at it amnd merge asap!

File: src/lib/ng-v5/entry-point/resources/stylesheet.transform.ts [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import * as fs from 'fs-extra';
 import * as path from 'path';
+import stripBom = require('strip-bom');
```
Comment (dherges) [inline src/lib/ng-v5/entry-point/resources/stylesheet.transform.ts]: I rebased your changes to latest changes on master and these four lines (3x stylesheets, 1x template) are affected. Right?

Comment (ahmedGomaa90) [reply src/lib/ng-v5/entry-point/resources/stylesheet.transform.ts]: yes that's right, aslo in package.json replaced `'src/**/*.spec.ts'` with `\"src/**/*.spec.ts\"` in `test:specs` script as it breaks on windows

Comment (dherges) [reply src/lib/ng-v5/entry-point/resources/stylesheet.transform.ts]: Good. Then let's get in!

**Pull request merged**